server/conf/evolutions/default/44.sql

```diff
@@ -0,0 +1,66 @@
+# --- Add database triggers to set the latest_status column on creating an application_event or
+# --- updating an application.
+# --- For rationale behind choosing database triggers, see:
+# ---   https://github.com/civiform/civiform/issues/3269#issuecomment-1232193163
+# --- Relevant documentation:
+# --- Trigger functions: https://www.postgresql.org/docs/current/sql-createtrigger.html
+# --- Create trigger statement: https://www.postgresql.org/docs/current/sql-createtrigger.html
+
+# --- !Ups
+
+alter table applications add column if not exists latest_status varchar;
+
+-- The latest status associated with an application is set to NULL if no application event entries
```

Comment:
Start the comment with what the method does, then have the exceptions/qualifiers second.   The comment kinda reads backwards in general to me, it ends with the intent and leads with the exception.

EG we should be specific that this returns the status text and not the status details json/ etc.

Reply:
Good call. Done!